src/node.h

```diff
@@ -350,7 +350,8 @@ NODE_DEPRECATED("Use ParseEncoding(isolate, ...)",
 })
 
 NODE_EXTERN void FatalException(v8::Isolate* isolate,
-                                const v8::TryCatch& try_catch);
+                                const v8::TryCatch& try_catch,
+                                bool should_catch = true);
```

Comment:
This is a semver-major (ABI-breaking) change because the types of all arguments are part of the name under which the function is exported … if you want to do this in a non-breaking manner, what you want is to remove the `= true` default, then create a function with the old signature in `node.cc` and make that a wrapper that explicitly passes `true`, or something along these lines.

Reply:
(Also, I’m not sure `should_catch` is something that *should* be exposed to addon authors?)

Reply:
Thanks for taking a look at this! That makes sense. Will refactor.